client/asset/interface.go

```diff
@@ -298,6 +308,19 @@ type NewAddresser interface {
 	NewAddress() (string, error)
 }
 
+// FeeRater is capable of retrieving a non-critical fee rate estimate for an
+// asset. SPV wallets, for example, cannot provide a fee rate estimate, so
```

Comment:
We might want to say "some SPV wallets" because it seems that eth is OK to provide estimates, right?  I thought it's tip oracle works and the base fee comes from block header.